### PR TITLE
Display invite notification to invitees

### LINKED ISSUE #9: Message not displayed for /invite'd player
There is an option in my messages.yml, InviteNotification, that supports a message being sent to the player who receives a /invite. I haven't been able to find anything by searching BigScary's repo for this code, and it doesn't currently work. 

I would love to see this implemented for showing a message to the recipient of an /invite. 

Here's the code I have:

```
  InviteNotification:
    Text: '{0} has invited you to visit!'
    Notes: '0: inviter''s name'
```

### PR DESCRIPTION
**Pull request by zedwick**
Re-added Messages.InviteNotification and Messages.InviteInstruction which were removed in 5.6.

Also added Messages.InviteAlreadySent as an optional variation on the InviteConfirmation message shown if the invitee has already been invited by the inviter. Default is the same as InviteConfirmation.
If invitee has already been invited, they are not shown any inviteNotification nor inviteInstruction.

Resolves #9

### CODE REVIEW

**Comment by RoboMWM** [comment]
Huh. Player#invitee only holds one inviter (i.e. can't be invited by multiple players)?

**Comment by zedwick** [comment]
Yeah, I was a little surprised by that. Guess it's just the most recent invite, presumably because you're expected to use your invite soon after receiving it anyway.

File: src/me/ryanhamshire/PopulationDensity/DataStore.java [hunk 1]
```diff
@@ -869,7 +869,10 @@ private void loadMessages()
         this.addDefault(defaults, Messages.RegionNamesOnlyLettersAndNumbers, "Region names may not include symbols or punctuation.", null);
         this.addDefault(defaults, Messages.RegionNameConflict, "There's already a region by that name.", null);
         this.addDefault(defaults, Messages.NoMoreRegions, "Sorry, you're in the only region.  Over time, more regions will open.", null);
+	this.addDefault(defaults, Messages.InviteAlreadySent, "{0} may now use /visit {1} to teleport to your home post.", "0: invitee's name, 1: inviter's name");
```
Comment (RoboMWM) [inline src/me/ryanhamshire/PopulationDensity/DataStore.java]: whitespace :(

If I ever decide to overhaul this plugin, I'll be sure to fix this lol.

**Review by RoboMWM — CHANGES_REQUESTED** [review]
Just fix the whitespace and we're good to go!

File: src/me/ryanhamshire/PopulationDensity/DataStore.java (hunk 1 above)
Comment (zedwick) [reply src/me/ryanhamshire/PopulationDensity/DataStore.java]: Oh, does this file using different tab spacing than the others? O.o

**Pull request merged**